### DISCUSSION TITLE
专辑分享『だから僕は音楽を辞めた』

### DISCUSSION THREAD

**Discussion by March-777th** [share121/inter-knot#4839] [其他]
![cover](https://github.com/user-attachments/assets/f04a5891-1412-4544-85e5-674e1e961a6f)
ヨルシカᐠ( ᑒ )ᐟ ヨルシカᐠ( ᑒ )ᐟ
[夸克网盘](https://pan.quark.cn/s/953250230402)
艺术家：ヨルシカ
专辑：だから僕は音楽を辞めた
作曲家：n-buna
音轨信息：48kHz 24bit

**Comment by HLXS-HLXS** [comment]
很有品味，这首歌我也喜欢<img src="https://bbs-static.miyoushe.com/static/2023/12/22/2a010db5d2ab94345ef4f69bad152122_3253943580882084522.png" alt="月下-爱你" width="75" >>

**Comment by Signedinn** [comment]
有品 我也很喜欢这首
![IMG_4267](https://github.com/user-attachments/assets/5154e11f-617e-4ff2-b548-50f7a6537e59)
